TypewiseAlert/TypewiseAlert.cs

```diff
@@ -2,63 +2,83 @@
 
 namespace TypewiseAlert
 {
-  public class TypewiseAlert
+  public class BatteryTemperatureAlert
   {
-    public enum BreachType {
+    const double PASSIVE_COOLING_LOWER_LIMIT=0;
+    const double HI_ACTIVE_COOLING_LOWER_LIMIT=0;
+    const double MED_ACTIVE_COOLING_LOWER_LIMIT=0;
+    const double PASSIVE_COOLING_UPPER_LIMIT=35;
+    const double HI_ACTIVE_COOLING_UPPER_LIMIT=45;
+    const double MED_ACTIVE_COOLING_UPPER_LIMIT=40;
+    
+    public enum BreachType 
+    {
       NORMAL,
       TOO_LOW,
       TOO_HIGH
     };
-    public static BreachType inferBreach(double value, double lowerLimit, double upperLimit) {
-      if(value < lowerLimit) {
+    
+    public static BreachType inferBreach(double value, double lowerLimit, double upperLimit) 
+    {
+      if(value < lowerLimit) 
+      {
         return BreachType.TOO_LOW;
       }
-      if(value > upperLimit) {
+      if(value > upperLimit) 
+      {
         return BreachType.TOO_HIGH;
       }
       return BreachType.NORMAL;
     }
-    public enum CoolingType {
+    
+    public enum CoolingType 
+    {
       PASSIVE_COOLING,
       HI_ACTIVE_COOLING,
       MED_ACTIVE_COOLING
     };
-    public static BreachType classifyTemperatureBreach(
-        CoolingType coolingType, double temperatureInC) {
-      int lowerLimit = 0;
-      int upperLimit = 0;
-      switch(coolingType) {
-        case CoolingType.PASSIVE_COOLING:
-          lowerLimit = 0;
-          upperLimit = 35;
-          break;
-        case CoolingType.HI_ACTIVE_COOLING:
-          lowerLimit = 0;
-          upperLimit = 45;
-          break;
-        case CoolingType.MED_ACTIVE_COOLING:
-          lowerLimit = 0;
-          upperLimit = 40;
-          break;
+    
+    public static BreachType classifyTemperatureBreach(CoolingType coolingType, double temperatureInC) 
+    {
+      double lowerLimit = PASSIVE_COOLING_LOWER_LIMIT;
+      double upperLimit = PASSIVE_COOLING_UPPER_LIMIT;
+      if(coolingType==CoolingType.HI_ACTIVE_COOLING)
```

Comment:
Cyclomatic complexity can be reduced here. Please think of ways to address this.